x-pack/legacy/plugins/fleet/public/pages/agent_list/components/enrollment_instructions/container.tsx

```diff
@@ -0,0 +1,10 @@
+/*
```

Comment:
Thoughts on using a static file to provide the instructions for these steps that is a handlebars template system?

This would make it easier for the Go developers working on the agent to PR tweaks as they need them made to the instructions vs having to ask Kibana / JS devs to tweak it?

Then we just display the contents of those files in EuiCode blocks?

Reply:
as a stopgap measure or as something to ship with? one big EuiCodeBlock is not a good UX imo. it also will not provide translation.

this PR is similar to the implementation done for Kibana's [Add Data](https://github.com/elastic/kibana/blob/master/src/legacy/core_plugins/kibana/common/tutorials/auditbeat_instructions.js#L25)

Reply:
No no, we would still have translated instructions much in the same way as your manual instruction that we will still need to add... but there will also be a large code block of YAML in each case with a few variables like the token that we will need to inject. Part of the instructions will be to copy/paste that large block into a file.

Reply:
oh I see, I misunderstood the meaning. it totally makes sense to move large yaml blocks out into a separate file rather than having it by line as in this PR. will work on that.

Reply:
I moved commands/config out into a [separate file](https://github.com/elastic/kibana/pull/48173/commits/7ca9738dfaf88c1f79567d0a962872f939d0b079#diff-8127a8bea40d9b485fdf25c726aa3ff8R1). they are just JS string constants rather than, for example, .yml files as we will have multiple commands/configs per platform, per type (shell/container/tools) and it seemed overkill to create one file each.

I added templating ability and tested by adding [`{config.enrollmentToken}`](https://github.com/elastic/kibana/pull/48173/commits/7ca9738dfaf88c1f79567d0a962872f939d0b079#diff-8127a8bea40d9b485fdf25c726aa3ff8R42), the value and name of it is configured [here](https://github.com/elastic/kibana/pull/48173/commits/7ca9738dfaf88c1f79567d0a962872f939d0b079#diff-b68127262760777837dda13838f864e2R64). just a test variable for now :)

what do you think?